watertap/data/techno_economic/ozone_aop.yaml

```diff
@@ -44,12 +67,32 @@ default:
     value: 5      
     units: kWh/lb
     reference: B. Mundy et al. (2018) doi.org/10.1080/01919512.2018.1467187 - Energy consumption 
-      for ozone generator, ozone destruct, nitrogen feed system, cooling water pumps, and other misc. energy uses. 
-  oxidant_dose:
-    value: 5
-    units: mg/L
+      for ozone generator, ozone destruct, nitrogen feed system, cooling water pumps, and other misc. energy uses.
 
 hydrogen_peroxide:
```

Comment:
Do we need to second sub-type here? And if so, does this mean  we should use "oxidant_to_ozone_ratio" instead of explicitly stating H2O2 in the parameter name below?

Reply:
Maybe for ozonation/aop, we only need hydrogen peroxide although I'm unaware of potential alternatives.

Reply:
OK, I am going to going with renaming the ratio then.